x/interchainqueries/types/registered_query.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+func (queryInfo *RegisteredQuery) GetOwnerAddress() (creator sdk.AccAddress, err error) {
+	creator, errCreator := sdk.AccAddressFromBech32(queryInfo.Owner)
+	return creator, sdkerrors.Wrapf(errCreator, ErrInvalidOwner.Error(), queryInfo.Owner)
+}
+
+func (queryInfo *RegisteredQuery) GetDepositCoin() (wager sdk.Coin) {
+	return sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(int64(queryInfo.Deposit)))
```

Comment:
`sdk.DefaultBondDenom` is not a proper constant here:
1. Because it's hardcoded as `stake`. We should use our own coin denom (`NTRN`) and we must have such constant somewhere in app.go;
2. We should not have **bond** denom at all, since Neutron doesn't have staking features

Reply:
We are going to have staking enabled for the testnet, let's keep it for now

Reply:
I decided to solve this problem from another side. We will store denom in the module param like other modules do.